src/components/30-organisms/footer/index.scss

```diff
@@ -0,0 +1,314 @@
+.o-footer {
+  text-align: initial;
+
+  color: $color-prim-white;
+  background-color: $color-prim-blue-dark-indigo;
+
+  border-top: 1px solid rgba(255, 255, 255, 0.25);
```

Comment:
undeclared color, shouldn't this be added to global colors after discussion with designers?

Reply:
Hmm, it's a white with .25% transparency, if it's an exception I think it's better to add it here. If not, yes then add it to globals as a `$white-transparent` or similar

Reply:
I would say we don't add it. It would, however, make sense to introduce the variable @jackblackCH is mentioning with 0.65 opacity, since that is quite common.